module/epochs/qc_client.go

```diff
@@ -127,9 +126,8 @@ func (c *QCContractClient) SubmitVote(ctx context.Context, vote *model.Vote) err
 func (c *QCContractClient) Voted(ctx context.Context) (bool, error) {
 
 	// execute script to read if voted
-	arg := jsoncdc.MustEncode(cadence.String(c.nodeID.String()))
 	template := templates.GenerateGetNodeHasVotedScript(c.env)
-	hasVoted, err := c.FlowClient.ExecuteScriptAtLatestBlock(ctx, template, []cadence.Value{cadence.String(arg)})
+	hasVoted, err := c.FlowClient.ExecuteScriptAtLatestBlock(ctx, template, []cadence.Value{cadence.String(c.nodeID.String())})
```

Comment:
What does `MustEncode` do? Just curious since you've removed it.

Reply:
It encodes the cadence value to a JSON representation. I removed it here because `ExecuteScriptAtLatestBlock` is already encoding the values (they were being double-encoded).